package.json

```diff
@@ -33,7 +35,7 @@
     "typescript": "^4.0.5"
   },
   "engines": {
-    "node": ">=8.9.0"
+    "node": ">=13.2.0"
   },
   "scripts": {
     "test": "nyc mocha",
```

Comment:
probably need to switch this over to something like c8, which uses a similar api, like i did on this one: https://github.com/TobiTenno/i18n-string-templates/blob/main/package.json#L12